src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -132,6 +132,7 @@ private BuildEventArgs ReadProjectImportedEventArgs()
             var fields = ReadBuildEventArgsFields();
             // Read unused Importance, it defaults to Low
             ReadInt32();
+            bool ignored = ReadBoolean();
```

Comment:
After https://github.com/Microsoft/msbuild/pull/2729 goes in, we need to surround this place with an if statement. This doesn't need to block on that PR but it's important to do it soon after.

Reply:
Basically 
```
bool ignored = false;

// the import ignored field was added in file format version 3
if (fileFormatVersion > 2)
{
    ignored = ReadBoolean();
}
...
e.ImportIgnored = ignored;
```